File: examples/src/Slides/Hyperlink/RemoveHyperlinks.cs
using Aspose.slides.Export

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
when the project is build. Please check https:#docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http:#www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http:#www.aspose.com/community/forums/default.aspx
*/

namespace Aspose.slides.Examples.CSharp.slides.Hyperlink
{
    class RemoveHyperlinks
    {
        public static void Run()
        {
            #ExStart:RemoveHyperlinks
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Slides_Presentations_Hyperlink()

            # Instantiate Presentation class
            Presentation presentation = new Presentation(dataDir + "Hyperlink.pptx")

            # Removing the hyperlinks from presentation
            presentation.HyperlinkQueries.RemoveAllHyperlinks()

            #Writing the presentation as a PPTX file
            presentation.save(dataDir + "RemovedHyperlink_out.pptx", slides.export.SaveFormat.PPTX)
            #ExEnd:RemoveHyperlinks
        }
    }
}

File: examples/src/Slides/Hyperlinks/remove_hyperlinks.py
import aspose.slides as slides


def remove_hyperlinks(global_opts):
    # Instantiate Presentation class
    with slides.Presentation(global_opts.data_dir + "hyperlink.pptx") as presentation:
        # Removing the hyperlinks from presentation
        presentation.hyperlink_queries.remove_all_hyperlinks()

        # Writing the presentation as a PPTX file
        presentation.save(global_opts.out_dir + "hyperlink_remove_all_hyperlinks_out.pptx",
                          slides.export.SaveFormat.PPTX)
